src/routes/(console)/project-[project]/auth/security/updateMockNumbers.svelte

```diff
@@ -32,7 +33,7 @@
         ? 'Available on Appwrite Cloud'
         : 'Upgrade to add mock phone numbers';
     let emptyStateDescription: string = isSelfHosted
-        ? 'Sign up to Cloud to add mock phone numbers to your projects.'
+        ? 'Sign up for Cloud to add mock phone numbers to your projects.'
```

Comment:
lets revert this change. the copy was already approved before

Reply:
I cross checked this with @sara-k-48 and made the change after approval.